src/contracts/actions.ts

```diff
@@ -27,12 +28,10 @@ export const overrideContractActions = (client: GenLayerClient<Transport, Simula
     account: Account;
     address: Address;
     functionName: string;
-    args: any[];
+    args: string[];
```

Comment:
I believe it should be CalldataEncodable, not a string (than plain js string will work nicely)